cmd/main.go

```diff
@@ -101,6 +101,13 @@ func main() {
                     Required:    true,
                     Destination: &dbPass,
                 },
+                &cli.StringFlag{
+                    Name:        "dbSchema",
+                    Aliases:     []string{"c", "C"},
+                    Usage:       "Define the schema of the database.",
+                    Required:    true,
+                    Destination: &dbSchema,
+                },
```

Comment:
Unfortunately I didn't fully understand how to add a default param for the StringFlag